feature-tests/apiEndpoints/endpoints.ts

```diff
@@ -0,0 +1,18 @@
+export default class EndPoints {
+  public static PRIVATE_API_GATEWAY_URL =
+    process.env.TEST_ENVIRONMENT === 'dev'
+      ? `https://${process.env.SAM_STACK_NAME}.execute-api.eu-west-2.amazonaws.com/${process.env.TEST_ENVIRONMENT}`
```

Comment:
I dont think we need to differentiate for dev here
`https://${process.env.CFN_PrivateApiGatewayId}.execute-api.eu-west-2.amazonaws.com/${process.env.TEST_ENVIRONMENT}`
Should work for all

Reply:
This method has been updated to use the .env value for the PRIVATE_API_GATEWAY_URL
removed === dev statement